src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -501,6 +503,15 @@ export default {
               case FilingTypes.TRANSITION:
                 this.loadTransitionFiling(filing)
                 break
+              case FilingTypes.REGISTRARS_NOTATION:
+                this.loadOtherFiling(filing, filing.registrarsNotation)
+                break
+              case FilingTypes.REGISTRARS_ORDER:
+                this.loadOtherFiling(filing, filing.registrarsOrder)
+                break
+              case FilingTypes.COURT_ORDER:
+                this.loadOtherFiling(filing, filing.courtOrder)
```

Comment:
So the default item bodies display according to the comps?

Reply:
It does a great job yea! Worked by default without much effort :) 

I do however having another commit which will apply some staff only properties conditional, so stay tuned, will require another review!